src/common/SPELLS/rogue.ts

```diff
@@ -573,6 +573,11 @@ const spells = spellIndexableList({
     name: 'Opportunity',
     icon: 'ability_rogue_pistolshot',
   },
+  AUDACITY: {
```

Comment:
```suggestion
  AUDACITY_BUFF: {
```

Reply:
I replaced this by `AUDACITY_TALENT_BUFF:` instead because I noticed other talents buffs being named this way, I hope that is correct